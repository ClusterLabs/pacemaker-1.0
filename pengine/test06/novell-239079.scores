Allocation scores:
clone_color: ms-drbd0 allocation score on xen-2: 0
clone_color: ms-drbd0 allocation score on xen-1: 0
clone_color: drbd0:0 allocation score on xen-2: 0
clone_color: drbd0:0 allocation score on xen-1: 0
clone_color: drbd0:1 allocation score on xen-2: 0
clone_color: drbd0:1 allocation score on xen-1: 0
native_color: drbd0:0 allocation score on xen-2: 0
native_color: drbd0:0 allocation score on xen-1: 0
native_color: drbd0:1 allocation score on xen-2: -1000000
native_color: drbd0:1 allocation score on xen-1: 0
drbd0:0 promotion score on xen-2: -1
drbd0:1 promotion score on xen-1: -1
native_color: fs_1 allocation score on xen-2: -1000000
native_color: fs_1 allocation score on xen-1: -1000000
drbd0:0 promotion score on xen-2: -1000000
drbd0:1 promotion score on xen-1: -1000000
