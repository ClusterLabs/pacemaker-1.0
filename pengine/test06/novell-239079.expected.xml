<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="33" num_updates="1020" dc-uuid="b292356d-d8af-4360-bcd3-423b3ec7f1be" have-quorum="true" remote-tls-port="0" validate-with="pacemaker-1.0">
   <configuration>
     <crm_config>
       <cluster_property_set id="cib-bootstrap-options"><nvpair id="nvpair.id21849" name="default-action-timeout" value="120s"/><nvpair id="nvpair.id21859" name="cluster-delay" value="60s"/><nvpair id="nvpair.id21868" name="symmetric-cluster" value="true"/><nvpair id="nvpair.id21877" name="stop-orphan-resources" value="true"/><nvpair id="nvpair.id21886" name="stop-orphan-actions" value="true"/><nvpair id="nvpair.id21896" name="remove-after-stop" value="false"/><nvpair id="nvpair.id21905" name="is-managed-default" value="true"/><nvpair id="nvpair.id21914" name="no-quorum-policy" value="ignore"/><nvpair id="nvpair.id21924" name="stonith-action" value="reboot"/><nvpair id="nvpair.id21933" name="stonith-enabled" value="0"/><nvpair id="nvpair.id21941" name="pe-error-series-max" value="100"/><nvpair id="nvpair.id21951" name="pe-warn-series-max" value="100"/><nvpair id="nvpair.id21960" name="pe-input-series-max" value="100"/><nvpair id="nvpair.id21969" name="default-resource-stickiness" value="100"/><nvpair id="nvpair.id21978" name="default-migration-threshold" value="0"/><nvpair id="nvpair.id21988" name="shutdown-escalation" value="5min"/><nvpair id="nvpair.id21997" name="startup-fencing" value="false"/><nvpair id="nvpair.id22006" name="last-lrm-refresh" value="1169747681"/></cluster_property_set>
     </crm_config>
     <nodes>
       <node id="b292356d-d8af-4360-bcd3-423b3ec7f1be" uname="xen-2" type="normal"/>
       <node id="00aaa02f-110f-44e6-9a9d-af93827bfdb9" uname="xen-1" type="normal"/>
     </nodes>
     <resources>
       <primitive id="fs_1" class="ocf" provider="heartbeat" type="Filesystem">
         <instance_attributes id="instance_attributes.id22053"><nvpair id="nvpair.id22060" name="fstype" value="reiserfs"/><nvpair value="/mnt/share1" id="nvpair.id22068" name="directory"/><nvpair id="nvpair.id22077" name="device" value="/dev/drbd0"/><nvpair name="target_role" id="nvpair.id22085" value="#default"/></instance_attributes>
       <meta_attributes id="primitive-fs_1.meta"><nvpair id="nvpair.meta.auto-71" name="target-role" value="#default"/></meta_attributes></primitive>
       <master id="ms-drbd0">
         <meta_attributes id="meta_attributes.id22101"><nvpair id="nvpair.id22108" name="clone-max" value="2"/><nvpair id="nvpair.id22116" name="clone-node-max" value="1"/><nvpair id="nvpair.id22124" name="master-max" value="1"/><nvpair id="nvpair.id22133" name="master-node-max" value="1"/><nvpair id="nvpair.id22141" name="notify" value="yes"/><nvpair id="nvpair.id22149" name="globally-unique" value="false"/><nvpair id="nvpair.id22158" name="target-role" value="#default"/></meta_attributes>
         <primitive id="drbd0" class="ocf" provider="heartbeat" type="drbd">
           <instance_attributes id="instance_attributes.id22179"><nvpair id="nvpair.id22186" name="drbd_resource" value="r0"/><nvpair id="nvpair.id22194" name="clone_overrides_hostname" value="no"/></instance_attributes>
         <meta_attributes id="primitive-drbd0.meta"/></primitive>
       <meta_attributes id="master_slave-ms-drbd0.meta"/></master>
     </resources>
     <constraints>
       <rsc_order id="drbd_before_fs" first="ms-drbd0" then="fs_1" then-action="start" first-action="promote" score="0"/>
       <rsc_colocation id="fs_on_drbd0" rsc="fs_1" with-rsc="ms-drbd0" with-rsc-role="Master" score="INFINITY"/>
       <rsc_order id="promote_after_demote" first="ms-drbd0" then="ms-drbd0" then-action="promote" first-action="demote" score="0"/>
     </constraints>
   </configuration>
   <status>
     <node_state id="b292356d-d8af-4360-bcd3-423b3ec7f1be" uname="xen-2" crmd="online" shutdown="0" in_ccm="true" ha="active" join="member" expected="member">
       <lrm id="b292356d-d8af-4360-bcd3-423b3ec7f1be">
         <lrm_resources id="lrm_resources.id22285">
           <lrm_resource id="fs_1" type="Filesystem" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="fs_1_monitor_0" operation="monitor" transition-key="3:0:a986d1e8-d3f7-40e9-a470-687924b34540" transition-magic="0:7;3:0:a986d1e8-d3f7-40e9-a470-687924b34540" call-id="2" crm_feature_set="1.0.8" rc-code="7" op-status="0" interval="0" op-digest="e38fc095cd72534897b21e24a8484122"/>
           </lrm_resource>
           <lrm_resource id="drbd0:0" type="drbd" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="drbd0-0_monitor_0" operation="monitor" transition-key="4:0:a986d1e8-d3f7-40e9-a470-687924b34540" transition-magic="0:7;4:0:a986d1e8-d3f7-40e9-a470-687924b34540" call-id="3" crm_feature_set="1.0.8" rc-code="7" op-status="0" interval="0" op-digest="7ed7a3f295075a81edf4174cf92234c6"/>
           </lrm_resource>
           <lrm_resource id="drbd0:1" type="drbd" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="drbd0-1_monitor_0" operation="monitor" transition-key="3:1:a986d1e8-d3f7-40e9-a470-687924b34540" transition-magic="0:7;3:1:a986d1e8-d3f7-40e9-a470-687924b34540" call-id="4" crm_feature_set="1.0.8" rc-code="7" op-status="0" interval="0" op-digest="7ed7a3f295075a81edf4174cf92234c6"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
       <transient_attributes id="b292356d-d8af-4360-bcd3-423b3ec7f1be">
         <instance_attributes id="instance_attributes.id22411"><nvpair id="nvpair.id22418" name="probe_complete" value="true"/></instance_attributes>
       </transient_attributes>
     </node_state>
     <node_state id="00aaa02f-110f-44e6-9a9d-af93827bfdb9" uname="xen-1" crmd="online" in_ccm="true" ha="active" join="member" expected="member" shutdown="0">
       <lrm id="lrm.auto-2">
         <lrm_resources id="lrm_resources.id22455">
           <lrm_resource id="fs_1" type="Filesystem" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="fs_1_monitor_0" operation="monitor" transition-key="6:0:a986d1e8-d3f7-40e9-a470-687924b34540" transition-magic="0:7;6:0:a986d1e8-d3f7-40e9-a470-687924b34540" call-id="2" crm_feature_set="1.0.8" rc-code="7" op-status="0" interval="0" op-digest="e38fc095cd72534897b21e24a8484122"/>
           </lrm_resource>
           <lrm_resource id="drbd0:0" type="drbd" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="drbd0-0_monitor_0" operation="monitor" transition-key="7:0:a986d1e8-d3f7-40e9-a470-687924b34540" transition-magic="0:7;7:0:a986d1e8-d3f7-40e9-a470-687924b34540" call-id="3" crm_feature_set="1.0.8" rc-code="7" op-status="0" interval="0" op-digest="7ed7a3f295075a81edf4174cf92234c6"/>
           </lrm_resource>
           <lrm_resource id="drbd0:1" type="drbd" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="drbd0-1_monitor_0" operation="monitor" transition-key="5:1:a986d1e8-d3f7-40e9-a470-687924b34540" transition-magic="0:7;5:1:a986d1e8-d3f7-40e9-a470-687924b34540" call-id="4" crm_feature_set="1.0.8" rc-code="7" op-status="0" interval="0" op-digest="7ed7a3f295075a81edf4174cf92234c6"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
       <transient_attributes id="transient_attributes.auto-2">
         <instance_attributes id="instance_attributes.id22577"><nvpair id="nvpair.id22584" name="probe_complete" value="true"/></instance_attributes>
       </transient_attributes>
     </node_state>
   </status>
 </cib>
